source/change-streams/change-streams.rst

```diff
@@ -776,3 +776,6 @@ Changelog
 +------------+------------------------------------------------------------+
 | 2019-06-20 | Fix server version for addition of postBatchResumeToken    |
 +------------+------------------------------------------------------------+
+| 2019-07-01 | Clarified that close may be implemented with more idiomatic|
```

Comment:
New change log entry seems like a good reason to bump the spec version. I believe it's currently 1.6.1. This is a minor clarification that neither changes or introduces functionality, so I think a patch bump to 1.6.2 is reasonable.

Reply:
Sounds good!